src/main/scala/abalone/format/Forsyth.scala

```diff
@@ -10,22 +10,22 @@ import strategygames.{Player, Score}
 object Forsyth {
   val initial = Abalone.initialFen // TODO?
 
-  def <<@(variant: Variant, fen: FEN): Option[Situation] = {
+  def <<@(variant: Variant, fen: FEN): Option[Situation] =
     Option(
       Situation(
         board = Board(
           pieces = fen.pieces(variant),
           history = History(
-            prevPlayer = fen.prevPlayer(variant),
+            lastTurn = fen.lastTurn(variant).fold(List[Uci]())(List(_)),
```

Comment:
Yeah so we don't need to rebuild the lastTurn from the FEN so I would omit this. Its ok not to have this information in the FEN